library/core/src/result.rs

```diff
@@ -1535,11 +1535,17 @@ impl<T, E> Result<&T, E> {
     /// ```
     #[inline]
     #[stable(feature = "result_copied", since = "1.59.0")]
-    pub fn copied(self) -> Result<T, E>
+    #[rustc_const_unstable(feature = "const_result", issue = "82814")]
+    pub const fn copied(self) -> Result<T, E>
     where
         T: Copy,
     {
-        self.map(|&t| t)
+        // FIXME: this implementation, which sidesteps using `Result::map` since it's not const
```

Comment:
i think we have a "fixme(const-hack)" term or something like that that's already used in many other places, if you find it can you use it here as well

Reply:
The comment here is copied verbatim from `Option::copied`.

We have a "const-hack" label on PRs, maybe you mean that? ("FIXME(const-hack)" sounds like a better idea than the label, though...)

Reply:
Ah, `FIXME(const-hack)` also exists. Interesting, somehow I missed that.